code/modules/client/preferences/species_features/sounds.dm

```diff
@@ -0,0 +1,59 @@
+/// Controls hearing dance machines
```

Comment:
This is inside the species_features folder, move it out to the other game preferences.